README.md

```diff
@@ -14,183 +14,315 @@
+| <p align="center">[Prepare the TinyLlama 1T token dataset](https://lightning.ai/lightning-ai/studios/prepare-the-tinyllama-1t-token-dataset) <br> [<img src="./images/3.webp" width="300"></p>](https://lightning.ai/lightning-ai/studios/prepare-the-tinyllama-1t-token-dataset) | [Pretrain LLMs - TinyLlama 1.1B](https://lightning.ai/lightning-ai/studios/pretrain-llms-tinyllama-1-1b) <br> <p align="center">[<img src="./images/4.webp" width="300"></p>](https://lightning.ai/lightning-ai/studios/pretrain-llms-tinyllama-1-1b) |
+| [Continued Pretraining with TinyLlama 1.1B](https://lightning.ai/lightning-ai/studios/continued-pretraining-with-tinyllama-1-1b) <br> <p align="center">[<img src="./images/1.webp" width="300"></p>](https://lightning.ai/lightning-ai/studios/continued-pretraining-with-tinyllama-1-1b) | [Instruction finetuning - TinyLlama 1.1B LLM](https://lightning.ai/lightning-ai/studios/instruction-finetuning-tinyllama-1-1b-llm) <br> <p align="center">[<img src="./images/2.webp" width="300"></p>](https://lightning.ai/lightning-ai/studios/instruction-finetuning-tinyllama-1-1b-llm) |
+|  |  |
 
-Avoiding code duplication is **not** a goal. **Readability** and **hackability** are.
 
-&nbsp;
 
-## Get involved!
 
-[Join our Discord](https://discord.gg/VptPCZkGNa) to build high-performance, truly open-source models for the common benefit of the community.
 
+&nbsp;
+<br>
 &nbsp;
 
-## Setup
 
-Install with all dependencies (including CLI, quantization, tokenizers for all models, etc.):
+
+## Installing LitGPT
+
+You can install LitGPT with all dependencies (including CLI, quantization, tokenizers for all models, etc.) using the following pip command:
 
 ```bash
 pip install 'litgpt[all]'
 ```
 
+Alternatively, can install litgpt from a cloned GitHub repository:
+
+```bash
+git clone https://github.com/Lightning-AI/litgpt/blob/wip/config_hub/finetune/llama-2-7b/full.yaml
+cd litgpt
+pip install ".[all]"
+```
+
+
 &nbsp;
 
-## Use the model
+## Getting started in 3 steps
 
-To generate text predictions, you need to download the model weights. **If you don't have them, check out our [guide](tutorials/download_model_weights.md).**
+Below is a minimal example to get started with the LitGPT command line interface (CLI), illustrating how to download a model, optionally finetune it using low-rank adaptation (LoRA), and start chatting with it:
 
-Run inference:
 
 ```bash
-litgpt generate base --prompt "Hello, my name is"
+# 1) Download a pretrained model
+litgpt download --repo_id mistralai/Mistral-7B-v0.1
+
+# 2) Optionally finetune the model
+litgpt finetune lora \
+  --checkpoint_dir checkpoints/mistralai/Mistral-7B-v0.1 \
+  --train.micro_batch_size 2 \
+  --lora_r 4 \
+  --precision bf16-true \
+  --data Alpaca2k \
+  --out_dir out/my-finetuned-model
+
+# 3) Chat with the model
+litgpt finetune chat \
+  --checkpoint_dir checkpoints/mistralai/Mistral-7B-v0.1 \
```

Comment:
This checkpoint dir will depend on whether the user chose to run the optional finetuning step above. 
Also, Mistral-7B-v0.1 is not a chat model so I don't expect it to behave well (or does it?). 

For these commands, I think we should decide if an end-to-end example is more important, or whether individual steps like above are better.

Reply:
Good point. Let's just use chat here and show finetuning further below.